modular_skyrat/modules/rr_opt_in/code/objective_item.dm

```diff
@@ -0,0 +1,20 @@
+/datum/objective_item
+	/// The opt in level all owners of the item must meet for this to be eligible as an objective target.
+	var/opt_in_level = RR_OPT_OUT
+
+/// Returns TRUE if we have no owners, or all owners's effective opt in level is above [opt_in_level]. FALSE otherwise.
+/datum/objective_item/proc/owner_opted_in()
+	if (!length(item_owner))
+		return TRUE
+	for (var/mob/living/player as anything in GLOB.player_list)
+		if ((player.mind?.assigned_role.title in item_owner) && player.stat != DEAD && !is_centcom_level(player.z)) // is an owner, copypasted from objective_items.dm owner_exists()
+			if (player.mind.get_effective_opt_in_level() < opt_in_level)
+				return FALSE
```

Comment:
Change the logic in this to utilize early `continue`s rather than nested `if` statements.